docs/tutorial.md

```diff
@@ -6,6 +6,18 @@ title: Tutorial
 # Your first Aragon app
 In this guide, we will walk you through creating your first Aragon app using [aragonOS](os-ref.md), [aragon.js](js-ref.md), [Aragon UI](ui-intro.md) and [Aragon CLI](https://github.com/aragon/aragon-cli).
 
+# Quickstart
+
+```
+npm i -g @aragon/cli
+aragon init foo.aragonpm.eth react
+cd foo
+npm install
```

Comment:
As explained https://github.com/aragon/aragon-cli/issues/148#issuecomment-399354226, `npm install` shouldn't be required as it is already performed by the init command.